### PR TITLE
Upgrade to lassie v0.4.3

### PR DESCRIPTION
**Pull request by hannahhoward**
uses temp dir in home directory (I think)

### CODE REVIEW

**Review by DiegoRBaquero — APPROVED** [review]

File: container/start.sh [hunk 1]
```diff
@@ -52,6 +52,7 @@ if [ "${LASSIE_ORIGIN:-}" != "" ]; then
     --event-recorder-url $LASSIE_EVENT_RECORDER_URL \
     --event-recorder-auth $LASSIE_EVENT_RECORDER_AUTH \
     --event-recorder-instance-id "$(cat /usr/src/app/shared/nodeId.txt)" \
+    --temp-dir /usr/src/app/shared
```
Comment (guanzo) [inline container/start.sh:55]: need trailing slash i think, same as previous lines

Comment (guanzo) [inline container/start.sh:55]:
```suggestion
    --temp-dir /usr/src/app/shared \
```

**Comment by DiegoRBaquero** [comment]
Let's wait for https://github.com/filecoin-saturn/L1-node/pull/229 to go out first

**Comment by guanzo** [comment]
LGTM

**Comment by guanzo** [comment]
@hannahhoward fyi lassie accepts `--tempdir` and not `--temp-dir`.

**Pull request merged**